make_style.sh

```diff
@@ -14,10 +14,14 @@ if ! [ -x "$(command -v autoprefixer)" ]; then
   exit 1
 fi
 
-FILES=(sass_processed/style.css sass_processed/martor-description.css)
-
 cd "$(dirname "$0")" || exit
+
+echo 'Creating default style...'
+cp resources/vars_default.scss resources/vars.scss
```

Comment:
Could we write this as a for loop? There's a lot in common between the light/dark themes.

Reply:
Done. The generated files are now:

```
resources/style.css
resources/martor-description.css (used for admin's martor preview)
resources/dark/style.css
resources/dark/martor-description.css (useless, because admin doesn't do dark mode)
```